image/spec/specs-go/v1/image_deprecated.go

```diff
@@ -0,0 +1,28 @@
+package v1
```

Comment:
Should this package be deprecated?

Reply:
Ah, yes, can do. I keep forgetting that a package _as a whole_ can also be deprecated. Will do